README.md

```diff
@@ -845,7 +845,7 @@ Table of Contents
   * [darklang.com](https://darklang.com/) - Hosted language combined with editor and infrastructure. Free during the beta, generous free tier planned after beta.
   * [Firebase](https://firebase.com) — Firebase helps you build and run successful apps. Free Spark Plan offers Authentication, Hosting, Firebase ML , Realtime Database,Cloud Storage,Testlab. A/B Testing, Analytics, App Distribution, App Indexing, Cloud Messaging (FCM), Crashlytics, Dynamic Links, In-App Messaging, Performance Monitoring, Predictions, and Remote Config are always-free.
   * [Flutter Flow](https://flutterflow.io) — Build your Flutter App UI without writing a single line of code. Also has a Firebase integration. Free plan includes full access to UI Builder and Free templates.
-  * [getstream.io](https://getstream.io/) — Build scalable news feeds and activity streams in a few hours instead of weeks, free for 3 million feed updates/month
+  * [getstream.io](https://getstream.io/) — Build scalable newsfeeds, activity streams, chat and messaging in a few hours instead of weeks
```

Comment:
There appear to have been some changes to the offering no? What is free now? Looks like the chat is infact now a 30 day trial while activity feed is offered free, if thats the case we should call this out in the description

Reply:
@ripienaar Feeds and Chat are free with the Maker Account (https://getstream.io/maker-account/) which is accessible via both pricing pages. 

> The new plan provides qualifying teams with full and free access to Stream’s Chat API (2k MAU) and Activity Feed API (9 million feed updates/month).

Full details here: https://getstream.io/blog/maker-account/

<img width="1376" alt="Screen Shot 2021-06-24 at 2 48 43 PM" src="https://user-images.githubusercontent.com/1328388/123330606-77fed000-d4fb-11eb-9817-c18024f7a7a9.png">

Reply:
nice, somehow missed that, thanks